test/e2e/config/digitalocean-ci.yaml

```diff
@@ -0,0 +1,108 @@
+# - from the CAPDO repository root, `make e2e` to build the vsphere provider image and run e2e tests.
+
+images:
+  # Use local dev images built source tree;
+  - name: gcr.io/k8s-staging-cluster-api/cluster-api-controller-amd64:v0.3.9
+    loadBehavior: tryLoad
+  - name: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller-amd64:v0.3.9
+    loadBehavior: tryLoad
+  - name: gcr.io/k8s-staging-cluster-api/kubeadm-control-plane-controller-amd64:v0.3.9
+    loadBehavior: tryLoad
+  - name: gcr.io/k8s-staging-cluster-api/capdo-manager:e2e
+    loadBehavior: mustLoad
+  - name: quay.io/jetstack/cert-manager-cainjector:v0.16.1
+    loadBehavior: tryLoad
+  - name: quay.io/jetstack/cert-manager-webhook:v0.16.1
+    loadBehavior: tryLoad
+  - name: quay.io/jetstack/cert-manager-controller:v0.16.1
+    loadBehavior: tryLoad
+
+providers:
+  - name: cluster-api
+    type: CoreProvider
+    versions:
+      - name: v0.3.9
+        # Use manifest from source files
+        value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.9/core-components.yaml"
+        type: "url"
+        replacements:
+          - old: "imagePullPolicy: Always"
+            new: "imagePullPolicy: IfNotPresent"
+          - old: "--enable-leader-election"
+            new: "--enable-leader-election=false"
+  - name: kubeadm
+    type: BootstrapProvider
+    versions:
+      - name: v0.3.9
+        # Use manifest from source files
+        value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.9/bootstrap-components.yaml"
+        type: "url"
+        replacements:
+          - old: "imagePullPolicy: Always"
+            new: "imagePullPolicy: IfNotPresent"
+          - old: "--enable-leader-election"
+            new: "--enable-leader-election=false"
+  - name: kubeadm
+    type: ControlPlaneProvider
+    versions:
+      - name: v0.3.9
+        # Use manifest from source files
+        value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.9/control-plane-components.yaml"
+        type: "url"
+        replacements:
+          - old: "imagePullPolicy: Always"
+            new: "imagePullPolicy: IfNotPresent"
+          - old: "--enable-leader-election"
+            new: "--enable-leader-election=false"
+  - name: digitalocean
+    type: InfrastructureProvider
+    versions:
+      - name: v0.4.0
+        # Use manifest from source files
+        value: ../../../config
+        replacements:
+          - old: gcr.io/k8s-staging-cluster-api-do/cluster-api-do-controller:master
+            new: gcr.io/k8s-staging-cluster-api/capdo-manager:e2e
+          - old: "imagePullPolicy: Always"
+            new: "imagePullPolicy: IfNotPresent"
+    files:
+      # Add a cluster template
+      - sourcePath: "../data/infrastructure-digitalocean/cluster-template.yaml"
+      - sourcePath: "../data/infrastructure-digitalocean/metadata.yaml"
+
+variables:
+  KUBERNETES_VERSION: "v1.17.2"
```

Comment:
Should we consider testing newer versions as well?

Reply:
Updated to `v1.17.11`.